legal-api/report-templates/certificateOfRestoration.html

```diff
@@ -35,7 +35,8 @@
                 <span class="bold-italic">{{business.legalName}}</span>
               {% elif restoration.type == 'limitedRestoration' %}
                 has on {{effective_date_time}} been restored for a period ending
-                <span class="bold-italic">{{restoration_expiry_date}}</span> under the {{entityAct}}.
+                <span class="bold-italic">{{restoration_expiry_date}}</span> under the {{entityAct}} as
+                <span class="bold-italic">{{business.legalName}}</span>
```

Comment:
@severinbeauvais This is not a bug. Just realized it was the requirement. Does the requirement change? 
Ref: https://preview.uxpin.com/306da47387722817e24fc77fd71476a1869a05fe#/pages/160343881/simulate/no-panels

Reply:
Good catch,

I've asked Mihai about this specific scenario, which isn't shown in the UI designs. (Ie, a limited restoration where the historical business name is different from the limited-active business name.)

Reply:
Do we need the same logic in limitedRestorationExtension and limitedRestorationToFull?

Reply:
Probably.

Reply:
I will check with Mihai

Reply:
@ketaki-deodhar , check with Mihai about the other 2 resto scenarios. If there are no more changes then go ahead and merge.